.travis.yml

```diff
@@ -1,29 +1,41 @@
-language: minimal
+---
+language: python
+dist: xenial
+os: linux
 
-matrix:
+cache:
+  directories:
+    - $HOME/geosinstall
+    - ~/.cache/pip
+
+jobs:
   include:
-  - os: linux  # 2017
-    env: PYTHON_VERSION="3.6" DEPS="numpy=1.13 geos=3.6"
-  - os: linux  # 2018
-    env: PYTHON_VERSION="3.7" DEPS="numpy=1.15 geos=3.7"
-  - os: linux  # 2019
-    env: PYTHON_VERSION="3.8" DEPS="numpy=1.17 geos=3.8"
-  - os: linux  # now
-    env: PYTHON_VERSION="3.8" CONDA_FORGE=1 DEPS="numpy geos"
-  - os: osx
-    env: PYTHON_VERSION="3.8" DEPS="numpy geos"
+    - python: "3.5"
+      env: GEOS_VERSION="3.5.2"
+    - python: "3.6"
+      env: GEOS_VERSION="3.6.4"
+    - python: "3.7"
+      env: GEOS_VERSION="3.7.3"
+    - python: "3.8"
+      env: GEOS_VERSION="3.8.1"
+    - python: "3.9"
+      env: GEOS_VERSION="3.8.1"
+    - python: "3.9-dev"
+      env: GEOS_VERSION="master"
+  allow_failures:
+    - python: "3.9-dev"
+      env: GEOS_VERSION="master"
 
 install:
-  - wget https://repo.anaconda.com/pkgs/misc/conda-execs/conda-latest-$TRAVIS_OS_NAME-64.exe -O conda.exe
-  - chmod +x conda.exe
-  - export CONDA_ALWAYS_YES=1
-  - if [ $RESTORE_FREE_CHANNEL ]; then ./conda.exe config --set restore_free_channel true; fi
-  - if [ $CONDA_FORGE ]; then ./conda.exe config --add channels conda-forge; fi
-  - ./conda.exe create --prefix $HOME/miniconda python=$PYTHON_VERSION conda pytest $DEPS
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - hash -r
+  - ./ci/travis/install_geos.sh
+  - cd $TRAVIS_BUILD_DIR
+  - pip install --disable-pip-version-check --upgrade pip
+  - pip install --upgrade wheel
```

Comment:
do we need wheel?

Reply:
This provides wheel in the case when pip needs to create whl files for dependencies if they are not on PyPI (I think). It's a relatively uncommon thing, but this happened a few days ago while rolling out Python 3.9 support [here](https://ci.appveyor.com/project/frsci/shapely/builds/36080775/job/18hc5h93tjxysw4b#L1636), which was fixed by https://github.com/Toblerity/Shapely/pull/1014